src/utils/DMRoomMap.ts

```diff
@@ -175,9 +179,9 @@ export default class DMRoomMap {
             .reduce((obj, r) => (obj[r.userId] = r.room) && obj, {});
     }
 
-    _getUserToRooms() {
+    _getUserToRooms(): {[key: string]: string[]} {
```

Comment:
this should be private and un-underscored

Reply:
oops, fixed